src/track/libheaptrack.cpp

```diff
@@ -184,7 +184,14 @@ FILE* createFile(const char* fileName)
     auto out = fopen(outputFileName.c_str(), "w");
     debugLog<VerboseOutput>("will write to %s/%p\n", outputFileName.c_str(), out);
     // we do our own locking, this speeds up the writing significantly
-    __fsetlocking(out, FSETLOCKING_BYCALLER);
+    if(out) {
+        __fsetlocking(out, FSETLOCKING_BYCALLER);
+    }
+    else {
+        debugLog<MinimalOutput>("failure in opening %sp\n", outputFileName.c_str(), out);
```

Comment:
use `fprintf(stderr, ` to ensure this is always being printed. Also you propbably wanted to print `out` via `%p` (your format string is wrong)